MAINTAINERS.yml

```diff
@@ -459,23 +459,36 @@ Display drivers:
     - "area: Display"
 
 Documentation:
+  status: odd fixes
+  collaborators:
+    - carlescufi
+    - nashif
+  files:
+    - doc/contribute/
+    - doc/develop/
+    - doc/introduction/
+    - doc/project/
+    - doc/releases/
+    - doc/security/
+    - README.rst
+  labels:
+    - "area: Documentation"
+
+Documentation Infrastructure:
   status: maintained
   maintainers:
     - carlescufi
```

Comment:
can you set yourself as the Documentation Infrastructure maintainer and add me as the Documentation maintainer?

Reply:
done